pvm.py

```diff
@@ -43,9 +43,13 @@ def parse_commands(*args):
 		pass
 	# TODO: Create another python file to control two display
 	if command=="file":
-		_logger.info("File set: %s", PEFIX_PATH + value)
+		_logger.info("File set: %s", PREFIX_PATH + value)
+		if IS_FILE_SET:
+			_logger.info("The file has already been set!")
+			if media is not None:
+					media.stop()
 		IS_FILE_SET = True
-		media = OMXPlayer(PEFIX_PATH + value, dbus_name='org.mpris.MediaPlayer2.omxplayer', args=['--loop'])
+		media = OMXPlayer(PREFIX_PATH + value, dbus_name='org.mpris.MediaPlayer2.omxplayer', args=['--win', "50,50,1390,1030"])
```

Comment:
Have you tested that the video can loop after you remove the loop option?

Reply:
I intend to remove the loop option to faciliate the test.

Reply:
I see, totally agree.